fix-point/fix-point.cabal

```diff
@@ -34,6 +34,8 @@ library
       base >=4.7 && <5
   default-language: Haskell2010
 
+ghc-options: -Wall
```

Comment:
`ghc-options` shouldn't start from the beginning of the line without any indentation, they should be part of `library` stanza